src/Hamiltonians/excitations.jl

```diff
@@ -28,7 +28,6 @@ See [`excitation`](@ref), [`OccupiedModeMap`](@ref).
 )
     M = num_modes(add)
     singlies = length(map)
```

Comment:
```suggestion
    singlies = length(map) # number of at least singly occupied modes
```

I'd like to keep this code readable, so suggest generously adding explanatory comments